### ISSUE TITLE
[Bug]: All photo previews disappears

### ISSUE THREAD

**Issue by Svampebob1** [nextcloud/photos#1822]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

I open the Photos app. All of the previews disappear.

Recently this happened when scrolling down through the photos, until I deleted a lot of my photos. The issue now occurs a second after opening the photos app.

### Steps to reproduce

1. Open the photos app and see all of the photos first appear and then disappear after a second.

### Expected behavior

See all the thumbnails for the photos.

### Installation method

Community Docker image

### Nextcloud Server version

26

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 22.2.3 to 22.2.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [x] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "htaccess.RewriteBase": "\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memory_limit": "4G",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "home.pludge.com",
            "192.168.0.35:8080"
        ],
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "26.0.1.1",
        "maintenance": false,
        "overwrite.cli.url": "https:\/\/home.pludge.com",
        "overwriteprotocol": "https",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "PLAIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "default_phone_region": "NO",
        "twofactor_enforced": "true",
        "twofactor_enforced_groups": [],
        "twofactor_enforced_excluded_groups": [],
        "loglevel": 2,
        "theme": "",
        "app_install_overwrite": [
            "sharerenamer",
            "richdocumentscode"
        ]
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.18.0
  - bruteforcesettings: 2.6.0
  - calendar: 4.3.4
  - circles: 26.0.0
  - cloud_federation_api: 1.9.0
  - comments: 1.16.0
  - contacts: 5.2.0
  - contactsinteraction: 1.7.0
  - dashboard: 7.6.0
  - dav: 1.25.0
  - federatedfilesharing: 1.16.0
  - federation: 1.16.0
  - files: 1.21.1
  - files_external: 1.18.0
  - files_pdfviewer: 2.7.0
  - files_rightclick: 1.5.0
  - files_sharing: 1.18.0
  - files_trashbin: 1.16.0
  - files_versions: 1.19.1
  - fileslibreofficeedit: 1.1.0
  - firstrunwizard: 2.15.0
  - groupfolders: 14.0.2
  - integration_openproject: 2.3.6
  - logreader: 2.11.0
  - lookup_server_connector: 1.14.0
  - mail: 3.1.1
  - maps: 1.0.2
  - nextcloud_announcements: 1.15.0
  - notes: 4.7.2
  - notifications: 2.14.0
  - oauth2: 1.14.0
  - password_policy: 1.16.0
  - photos: 2.2.0
  - previewgenerator: 5.3.0
  - privacy: 1.10.0
  - provisioning_api: 1.16.0
  - recognize: 4.0.1
  - recommendations: 1.5.0
  - related_resources: 1.1.0-alpha1
  - richdocuments: 8.0.2
  - richdocumentscode: 22.5.1301
  - serverinfo: 1.16.0
  - settings: 1.8.0
  - sharebymail: 1.16.0
  - sharerenamer: 3.2.0
  - spreed: 16.0.3
  - support: 1.9.0
  - survey_client: 1.14.0
  - suspicious_login: 4.4.0
  - systemtags: 1.16.0
  - tasks: 0.15.0
  - text: 3.7.2
  - theming: 2.1.1
  - twofactor_backupcodes: 1.15.0
  - twofactor_nextcloud_notification: 3.7.0
  - twofactor_totp: 8.0.0-alpha.0
  - updatenotification: 1.16.0
  - user_status: 1.6.0
  - viewer: 1.10.0
  - weather_status: 1.6.0
  - workflowengine: 2.8.0
Disabled:
  - admin_audit: 1.16.0
  - encryption: 2.14.0
  - group_default_quota: 0.1.6 (installed 0.1.6)
  - user_ldap: 1.16.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Log entries that might be related to the issue

```nextcloud log entries that might be related
{"reqId":"yn9XM43adrW0JWnx032u","level":3,"time":"2023-05-04T06:58:38+00:00","remoteAddr":"82.146.64.12","user":"--","app":"index","method":"GET","url":"/apps/files/api/v1/stats","message":"OCA\\Files\\Service\\TagService::__construct(): Argument #4 ($homeFolder) must be of type OCP\\Files\\Folder, null given, called in /var/www/html/apps/files/lib/AppInfo/Application.php on line 111","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","exception":{"Exception":"TypeError","Message":"OCA\\Files\\Service\\TagService::__construct(): Argument #4 ($homeFolder) must be of type OCP\\Files\\Folder, null given, called in /var/www/html/apps/files/lib/AppInfo/Application.php on line 111","Code":0,"Trace":[{"file":"/var/www/html/apps/files/lib/AppInfo/Application.php","line":111,"function":"__construct","class":"OCA\\Files\\Service\\TagService","type":"->","args":[["OC\\User\\Session"],["OC\\Activity\\Manager"],null,null,["OC\\EventDispatcher\\SymfonyAdapter"]]},{"file":"/var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php","line":171,"function":"OCA\\Files\\AppInfo\\{closure}","class":"OCA\\Files\\AppInfo\\Application","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/pimple/pimple/src/Pimple/Container.php","line":122,"function":"OC\\AppFramework\\Utility\\{closure}","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php","line":138,"function":"offsetGet","class":"Pimple\\Container","type":"->","args":["OCA\\Files\\Service\\TagService"]},{"file":"/var/www/html/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":487,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["OCA\\Files\\Service\\TagService"]},{"file":"/var/www/html/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->","args":["OCA\\Files\\Service\\TagService"]},{"file":"/var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php","line":65,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->","args":["OCA\\Files\\Service\\TagService"]},{"file":"/var/www/html/apps/files/lib/AppInfo/Application.php","line":90,"function":"get","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["OCA\\Files\\Service\\TagService"]},{"file":"/var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php","line":171,"function":"OCA\\Files\\AppInfo\\{closure}","class":"OCA\\Files\\AppInfo\\Application","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/3rdparty/pimple/pimple/src/Pimple/Container.php","line":122,"function":"OC\\AppFramework\\Utility\\{closure}","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php","line":138,"function":"offsetGet","class":"Pimple\\Container","type":"->","args":["APIController"]},{"file":"/var/www/html/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":487,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["APIController"]},{"file":"/var/www/html/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->","args":["APIController"]},{"file":"/var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php","line":65,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->","args":["APIController"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":148,"function":"get","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["APIController"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["APIController","getStorageStats","*** sensitive parameters replaced ***",["files.API.getStorageStats"]]},{"file":"/var/www/html/lib/base.php","line":1056,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/files/api/v1/stats"]},{"file":"/var/www/html/index.php","line":36,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/apps/files/lib/Service/TagService.php","Line":52,"CustomMessage":"--"}}
{"reqId":"bNkXfx9ZObK1mXLj6xlh","level":3,"time":"2023-05-06T08:01:41+00:00","remoteAddr":"82.146.65.146","user":"oystein","app":"index","method":"GET","url":"/apps/photos/api/v1/preview/1179198?etag=64c8b1fd16130db9abefe6b128878259&x=512&y=512","message":"Could not create folder","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","exception":{"Exception":"OCP\\Files\\NotPermittedException","Message":"Could not create folder","Code":0,"Trace":[{"file":"/var/www/html/lib/private/Files/AppData/AppData.php","line":147,"function":"newFolder","class":"OC\\Files\\Node\\Folder","type":"->","args":["a/6/1/4/1/0/8/1179198"]},{"file":"/var/www/html/lib/private/Preview/Storage/Root.php","line":74,"function":"newFolder","class":"OC\\Files\\AppData\\AppData","type":"->","args":["a/6/1/4/1/0/8/1179198"]},{"file":"/var/www/html/lib/private/Preview/Generator.php","line":685,"function":"newFolder","class":"OC\\Preview\\Storage\\Root","type":"->","args":["1179198"]},{"file":"/var/www/html/lib/private/Preview/Generator.php","line":139,"function":"getPreviewFolder","class":"OC\\Preview\\Generator","type":"->","args":[["OC\\Files\\Node\\File"]]},{"file":"/var/www/html/lib/private/Preview/Generator.php","line":116,"function":"generatePreviews","class":"OC\\Preview\\Generator","type":"->","args":[["OC\\Files\\Node\\File"],[[512,512,false,"fill"]],"image/jpeg"]},{"file":"/var/www/html/lib/private/PreviewManager.php","line":192,"function":"getPreview","class":"OC\\Preview\\Generator","type":"->","args":[["OC\\Files\\Node\\File"],512,512,false,"fill",null]},{"file":"/var/www/html/apps/photos/lib/Controller/PreviewController.php","line":162,"function":"getPreview","class":"OC\\PreviewManager","type":"->","args":[["OC\\Files\\Node\\File"],512,512]},{"file":"/var/www/html/apps/photos/lib/Controller/PreviewController.php","line":128,"function":"fetchPreview","class":"OCA\\Photos\\Controller\\PreviewController","type":"->","args":[["OC\\Files\\Node\\File"],512,512]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"index","class":"OCA\\Photos\\Controller\\PreviewController","type":"->","args":[1179198,512,512]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Photos\\Controller\\PreviewController"],"index"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":183,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Photos\\Controller\\PreviewController"],"index"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Photos\\Controller\\PreviewController","index",["OC\\AppFramework\\DependencyInjection\\DIContainer"],["1179198","photos.preview.index"]]},{"file":"/var/www/html/lib/base.php","line":1056,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/photos/api/v1/preview/1179198"]},{"file":"/var/www/html/index.php","line":36,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/lib/private/Files/Node/Folder.php","Line":166,"CustomMessage":"--"}}
{"reqId":"fhg2HJniiInhLLYQ3OgR","level":3,"time":"2023-05-06T08:01:41+00:00","remoteAddr":"82.146.65.146","user":"oystein","app":"index","method":"GET","url":"/apps/photos/api/v1/preview/1179190?etag=62e2589fb0129c6bcb22e1e6fbc6ce4d&x=64&y=64","message":"Could not create folder","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","exception":{"Exception":"OCP\\Files\\NotPermittedException","Message":"Could not create folder","Code":0,"Trace":[{"file":"/var/www/html/lib/private/Files/AppData/AppData.php","line":147,"function":"newFolder","class":"OC\\Files\\Node\\Folder","type":"->","args":["4/5/3/1/7/7/7/1179190"]},{"file":"/var/www/html/lib/private/Preview/Storage/Root.php","line":74,"function":"newFolder","class":"OC\\Files\\AppData\\AppData","type":"->","args":["4/5/3/1/7/7/7/1179190"]},{"file":"/var/www/html/lib/private/Preview/Generator.php","line":685,"function":"newFolder","class":"OC\\Preview\\Storage\\Root","type":"->","args":["1179190"]},{"file":"/var/www/html/lib/private/Preview/Generator.php","line":139,"function":"getPreviewFolder","class":"OC\\Preview\\Generator","type":"->","args":[["OC\\Files\\Node\\File"]]},{"file":"/var/www/html/lib/private/Preview/Generator.php","line":116,"function":"generatePreviews","class":"OC\\Preview\\Generator","type":"->","args":[["OC\\Files\\Node\\File"],[[64,64,false,"fill"]],"image/jpeg"]},{"file":"/var/www/html/lib/private/PreviewManager.php","line":192,"function":"getPreview","class":"OC\\Preview\\Generator","type":"->","args":[["OC\\Files\\Node\\File"],64,64,false,"fill",null]},{"file":"/var/www/html/apps/photos/lib/Controller/PreviewController.php","line":162,"function":"getPreview","class":"OC\\PreviewManager","type":"->","args":[["OC\\Files\\Node\\File"],64,64]},{"file":"/var/www/html/apps/photos/lib/Controller/PreviewController.php","line":128,"function":"fetchPreview","class":"OCA\\Photos\\Controller\\PreviewController","type":"->","args":[["OC\\Files\\Node\\File"],64,64]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"index","class":"OCA\\Photos\\Controller\\PreviewController","type":"->","args":[1179190,64,64]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Photos\\Controller\\PreviewController"],"index"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":183,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[["OCA\\Photos\\Controller\\PreviewController"],"index"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Photos\\Controller\\PreviewController","index",["OC\\AppFramework\\DependencyInjection\\DIContainer"],["1179190","photos.preview.index"]]},{"file":"/var/www/html/lib/base.php","line":1056,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/photos/api/v1/preview/1179190"]},{"file":"/var/www/html/index.php","line":36,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/lib/private/Files/Node/Folder.php","Line":166,"CustomMessage":"--"}}
{"reqId":"Lhcge44yJiGARUJ40HvM","level":3,"time":"2023-05-10T12:25:52+00:00","remoteAddr":"82.146.65.146","user":"oystein","app":"PHP","method":"SEARCH","url":"/remote.php/dav","message":"Allowed memory size of 536870912 bytes exhausted (tried to allocate 16384 bytes) at /var/www/html/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php#101","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.24.2","version":"26.0.1.1","data":{"app":"PHP"}}
{"reqId":"KMwidJ9cUYP1oGZh7nhM","level":3,"time":"2023-05-11T06:50:01+00:00","remoteAddr":"82.146.65.146","user":"oystein","app":"index","method":"GET","url":"/apps/photos/api/v1/preview/1179267?etag=b4c044dafb29e500db0f8a8034a11d7f&x=512&y=512","message":"Connection refused","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","exception":{"Exception":"RedisException","Message":"Connection refused","Code":0,"Trace":[{"file":"/var/www/html/lib/private/RedisFactory.php","line":137,"function":"pconnect","class":"Redis","type":"->","args":["127.0.0.1",6379,0,null,0,0]},{"file":"/var/www/html/lib/private/RedisFactory.php","line":178,"function":"create","class":"OC\\RedisFactory","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/lib/private/Memcache/Redis.php","line":50,"function":"getInstance","class":"OC\\RedisFactory","type":"->","args":[]},{"file":"/var/www/html/lib/private/Memcache/Redis.php","line":56,"function":"getCache","class":"OC\\Memcache\\Redis","type":"->","args":[]},{"file":"/var/www/html/lib/private/Memcache/WithLocalCache.php","line":24,"function":"get","class":"OC\\Memcache\\Redis","type":"->","args":["5dd6cd86d6cbf12ddd3ba8d5d83695f0361010d6"]},{"file":"/var/www/html/lib/private/User/Manager.php","line":169,"function":"get","class":"OC\\Memcache\\WithLocalCache","type":"->","args":["5dd6cd86d6cbf12ddd3ba8d5d83695f0361010d6"]},{"file":"/var/www/html/lib/private/User/Session.php","line":228,"function":"get","class":"OC\\User\\Manager","type":"->","args":["oystein"]},{"file":"/var/www/html/lib/private/legacy/OC_App.php","line":401,"function":"getUser","class":"OC\\User\\Session","type":"->","args":[]},{"file":"/var/www/html/lib/private/legacy/OC_App.php","line":123,"function":"getEnabledApps","class":"OC_App","type":"::","args":[]},{"file":"/var/www/html/lib/private/legacy/OC_User.php","line":128,"function":"loadApps","class":"OC_App","type":"::","args":[["prelogin"]]},{"file":"/var/www/html/lib/base.php","line":758,"function":"setupBackends","class":"OC_User","type":"::","args":[]},{"file":"/var/www/html/lib/base.php","line":1165,"function":"init","class":"OC","type":"::","args":[]},{"file":"/var/www/html/index.php","line":34,"args":["/var/www/html/lib/base.php"],"function":"require_once"}],"File":"/var/www/html/lib/private/RedisFactory.php","Line":137,"CustomMessage":"--"}}
{"reqId":"SY88jzeaFmNt7DAIKOn6","level":3,"time":"2023-05-11T06:50:01+00:00","remoteAddr":"82.146.65.146","user":"oystein","app":"core","method":"GET","url":"/apps/photos/api/v1/preview/1179212?etag=0ea9e6a0694da6a83299f76386cdca5d&x=512&y=512","message":"Redis server went away","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0","version":"26.0.1.1","exception":{"Exception":"RedisException","Message":"Redis server went away","Code":0,"Trace":[{"file":"/var/www/html/lib/private/Memcache/Redis.php","line":56,"function":"get","class":"Redis","type":"->","args":["1e6a0f83b7d2ff066a0768667e725de4/user_backend_map5dd6cd86d6cbf12ddd3ba8d5d83695f0361010d6"]},{"file":"/var/www/html/lib/private/Memcache/WithLocalCache.php","line":24,"function":"get","class":"OC\\Memcache\\Redis","type":"->","args":["5dd6cd86d6cbf12ddd3ba8d5d83695f0361010d6"]},{"file":"/var/www/html/lib/private/User/Manager.php","line":169,"function":"get","class":"OC\\Memcache\\WithLocalCache","type":"->","args":["5dd6cd86d6cbf12ddd3ba8d5d83695f0361010d6"]},{"file":"/var/www/html/lib/private/User/Session.php","line":228,"function":"get","class":"OC\\User\\Manager","type":"->","args":["oystein"]},{"file":"/var/www/html/lib/private/L10N/Factory.php","line":199,"function":"getUser","class":"OC\\User\\Session","type":"->","args":[]},{"file":"/var/www/html/lib/public/Util.php","line":227,"function":"findLanguage","class":"OC\\L10N\\Factory","type":"->","args":["core"]},{"file":"/var/www/html/lib/private/legacy/OC_Template.php","line":111,"function":"addTranslations","class":"OCP\\Util","type":"::","args":["core"]},{"file":"/var/www/html/lib/private/legacy/OC_Template.php","line":77,"function":"initTemplateEngine","class":"OC_Template","type":"::","args":["error"]},{"file":"/var/www/html/lib/private/legacy/OC_Template.php","line":321,"function":"__construct","class":"OC_Template","type":"->","args":["","exception","error",false]},{"file":"/var/www/html/index.php","line":74,"function":"printExceptionErrorPage","class":"OC_Template","type":"::","args":[["RedisException"],500]}],"File":"/var/www/html/lib/private/Memcache/Redis.php","Line":56,"CustomMessage":"--"}}
```

### Additional info

This is the official Nextcloud Docker image. 
I'm running it behind Nginx Proxy Manager as reverse proxy.

It seems to me that the problem might have happened due to lack of memory. I have extended the memory from the default in docker to 4G, since the problem occured.

**Comment by Svampebob1** [comment]
I highly suspect the problem is related to corrupt previews or images. Does anyone have any tip on how to locate the images that creates the problem?

**Comment by Svampebob1** [comment]
> There is a similar issue open (I was looking after the exact same issue I ran into today) and a fix is promised for 26.0.2 if I'm correct. A workaround until 26.0.2 is released is to pick the js folder out of the new release and copy in place of the current under photos, but I can't advocate for doing that, this is just what I've read. I also have Memories installed and everything looks fine in there, only Photos seems to be affected.
> 
> Here's the issue that this is (I think) similar to: #1753
> 
> 26.0.2 is scheduled for the 25th of May (in 2 days) by the way. https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule

Thank you! Then I'll just wait for a fix :-).
